### ISSUE TITLE
rig name conflict with existing command

### ISSUE THREAD

**Issue by deltaray** [TurboTurtle/rig#37]
Hi. I wanted to let you know about a name conflict with your command. I have nothing to do with it, but there is already a program called 'rig' that is the Random Identity Generator and has existed since 1999. In the Debian repo it is also called 'rig'. This already created some confusion when talking about the 'rig' command. I'm not suggesting a name change because it wouldn't be the first time there has been conflicting command names. However, since your project is much newer, I thought you might be open to the idea. Thanks.

**Comment by TurboTurtle** [comment]
Hi, thanks for the heads up.

I'm curious as to the conversations around this `rig` thus far in the Debian circles - so far this has been a pet project of mine with the RH support folks. More visibility/contributors is welcome of course, as is the prospect of shipping in more distributions :).

We're already shipping on Fedora as `rig` however, and that is the name we'll be shipping under RHEL. I'm not all that familiar with packaging for `.deb`s myself, so I'm curious as to if this can be solved during that specific packaging process. The `sos` project does something similar - on RH distros the package ships as `sos`, whereas on Debian/Ubuntu it ships as `sosreport`. Is that feasible here?

**Comment by deltaray** [comment]
Thanks for responding. I don't have anything to do with the packages or the programs so can't comment on that. I'm just a user. It came up this time when I posted this to climagic: https://twitter.com/climagic/status/1338469684002349061
The earliest time I posted about this command was in 2012: https://twitter.com/climagic/status/231400206405615616
Someone else told me about it sometime before that, so it seems to be used. I guess to be fair, my concern is in the confusion over the command name itself rather than the package name. Well, I think I've helped just by letting you know about the issue. It's not really an urgent thing, just wanted to give you an FYI.
